### ISSUE TITLE
Feature request: Ping on my personal @-mention

### ISSUE THREAD

**Issue by markostar** [integrations/microsoft-teams#52]
It would be awesome to be pinged on Teams for every new @-mention within my pre-specified Repos or orgs - providing a link, or a possibility to respond directly from Teams 

Thank you 
❤️ :octocat:

**Comment by ashokirla** [comment]
@markostar Yes, we are currently working on personal notifications feature. If you are interested, we would like to  share our plan and understand your requirements. Can you lets us know if it is possible to have a quick sync up.

**Comment by jonadaly** [comment]
This feature would be a massive improvement. PullPanda, which Github acquired last year, supported this by allowing admins to hook up slack usernames with github usernames.

Alternatively, the new Github/Slack integration also supports people being individually messaged when their review is requested - could the same feature be built here?

**Comment by peter-lillevold-4ss** [comment]
Yes, please! This is currently our number one missing feature. We would like to see:
* when a PR is assigned to a reviewer, the reviewer would be @'ed in Teams
* when GitHub users are mentioned in PR comments, the corresponding people would be @'ed in Teams

**Comment by aaron-kruse** [comment]
Any update on this? It looks like #80 and #132 are both related to this but possibly slightly different variations. The personal notifications previously mentioned might help in some situations, but not all.

My specific scenario is we have a channel setup in Teams that basically corresponds with our GitHub repo. I've set up [Scheduled Reminders](https://github.com/integrations/microsoft-teams/blob/main/Readme.md#schedule-reminders) so reminders about PRs pending review/merge will be posted to this channel in Teams. I personally have my notifications setup so I'll get a notification for new posts in this channel, but others do not get notified of all posts. For this reason, I wanted the reminders to include an @-mention to ensure they get a notification regardless of each user's channel notification settings.

From the [screenshots](https://github.com/integrations/microsoft-teams/blob/main/images/ScheduleReminders.PNG) in the documentation, this appears to be the expected behavior (ex. "Waiting on @amandabrady" with "@amandabrady" appearing as a linked @-mention). Once setup, however, it actually comes through as "Waiting on @github-user" and not "Waiting on @teams-user". Maybe this works out if the user in Teams and GitHub both happen to be the same, but I'm guessing this is not the case for most people. Without a way to "link" Teams and GitHub names to get "real" mentions, this kind of defeats the purpose for us (I can't ensure mention-level notifications regardless of how channel notifications are setup for each user).

**Comment by uridium** [comment]
@aaron-kruse I tried that tag trick, but it doesn't work for me. Are tags treated the same way as personal mentions?

My Teams notifications settings:


1:
![image](https://github.com/integrations/microsoft-teams/assets/15939365/20ae1868-01da-4698-9059-ab7422c72cc8)

2:
![image](https://github.com/integrations/microsoft-teams/assets/15939365/ecce783d-d940-4e18-bbbf-3b291d9dab63)


@ashokirla I see this issue as "completed". Could you point me, where could I read on how to notify someone directly on Teams, if they are assigned as reviewers? Also, my GitHub users have different names than users in MS Teams (Active Directory), what makes it more tricky. Here, in GitHub I use "uridium" name, when in MS Teams, I'm logged in as "firstname.lastname@domain.com".

**Comment by aaron-kruse** [comment]
> @aaron-kruse I tried that tag trick, but it doesn't work for me. Are tags treated the same way as personal mentions?

Yes, that seems to be the case, although the actual behavior is kind of hit or miss. I haven't dug too far into it, but it seems like it was initially working as expected. Later one specific person started having problems where it would sometimes work but not other times. More recently it seems like it never works for that person anymore. At the same time, however, it's still working fine for other people (ex. if a PR is waiting on approval from that person and someone else, that person's name/tag is regular text while the other person's is a clickable link).
